pkg/sql/drop_test.go

```diff
@@ -721,7 +721,7 @@ func TestDropTableDeleteData(t *testing.T) {
 	sqlRun := sqlutils.MakeSQLRunner(sqlDB)
 	for i := 0; i < numTables; i++ {
 		if err := descExists(sqlDB, true, descs[i].GetID()); err != nil {
-			t.Fatal(err)
+			t.Fatalf("table (id=%d name=%s) was deleted. error: %v", descs[i].GetID(), descs[i].GetName(), err)
```

Comment:
nit: maybe use `errors.Wrapf` here too

Reply:
since we're logging the failure right here and aborting the test, i don't think there's any need to wrap the error (which is usually used for propagating the error higher up the stack).